File: 프로그래머스/0/181839. 주사위 게임 1/주사위 게임 1.js
function solution(a, b) {
     if(a % 2 === 1 && b % 2 === 1) {
         return a**2 + b**2
     } else if ((a % 2 === 1 && b % 2 === 0) || (a % 2 === 0 && b % 2 === 1)) {
         return 2 *(a+b)
    } else {
        return Math.abs(a-b)
    }
}

File: 프로그래머스/0/181839. 주사위 게임 1/주사위 게임 1.py
def solution(a, b):
    if a % 2 == 1 and b % 2 == 1:
        return a**2 + b**2
    elif (a + b) % 2 == 1:
        return 2 * (a + b)
    else:
        return abs(a - b)